app/helpers/application_helper.rb

```diff
@@ -60,10 +60,4 @@ def cancel_link_text
   def desktop_device?
     !BrowserCache.parse(request.user_agent).mobile?
   end
-
-  def selfie_status_from_response(client_response)
```

Comment:
I think that this is okay to remove because we're no longer supporting the Acuant response as well?

Reply:
I'm seeing a lot of changes related to this. Can we do this in another PR or is there a reason it's better/easier to do it here?

Reply:
@charleyf - i am comfortable with making the change here.  is there a  big concern that I overlooking with this change in this PR?

Reply:
> is there a big concern that I overlooking with this change in this PR?

I don't have a big concern about this change specifically. It's that the more independent changes being made in a PR, the harder it is for me to figure out exactly what that PR is doing.

Could you tell me more about the reason for changing `selfie_status_from_response`? I feel like I remember some earlier discussion on moving this, but I can't find it right now.

Reply:
@charleyf 
1. for consistency, this apprach is in accord with our similarly used method (ie: [:doc_auth_sucess?](https://github.com/18F/identity-idp/blob/0b6e886f0e49afe3bf068f25bfaf0499ff01e410/app/services/doc_auth/response.rb#L102)) which I also find simpler. Also, this also makes room for vendor specific implementation should there be others in the future
2. there was also[ expressed concern](https://github.com/18F/identity-idp/pull/9953#discussion_r1465410755) about having a generic helper that is only relevant in DocAuth::Response
3. the places in which this method is used already have the doc_auth_response instance and does not required the use of a helper

Reply:
Ah, thanks! That makes sense, so basically this code doesn't need to be in `application_helper.rb` and moving it out is safe to do and cleans things up.